src/main/webapp/WEB-INF/views/allCars.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <title>Display all cars</title>
+</head>
+<body>
+<h1>All cars</h1>
+<form method="get" action="${pageContext.request.contextPath}/cars/drivers/delete" id="delete" name="form">
+    <select size="3" form="delete" id="select" name="id">
+        <c:forEach items="${cars}" var="car">
+            <option value="${car.id}"/>${car.id} ${car.model} ${car.manufacturer.id} ${car.manufacturer.name}
+             ${car.manufacturer.country}
+            <c:forEach items="${car.drivers}" var="driver">${driver.id} ${driver.name} ${driver.licenseNumber}</c:forEach>
+            </option>
+        </c:forEach>
+    </select>
+    <button type="submit">Delete</button>
+</form>
+
+</table>
+<a href="${pageContext.request.contextPath}/index">Main</a>
+<a href="${pageContext.request.contextPath}/cars/create">Add car</a>
+<a href="${pageContext.request.contextPath}/manufacturers/create">Add manufacturer</a>
+<a href="${pageContext.request.contextPath}/drivers/add">Add driver</a>
+<a href="${pageContext.request.contextPath}/cars/drivers/add">Add driver to car</a>
+</body>
+</html>
```

Comment:
There should be a blank line at the end of the file. Fix it everywhere

Reply:
done